image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
On Fri, Jul 22, 2016 at 08:09:47AM -0700, Antonio Murdaca wrote:
> the point now is that, unfortunately this tool is broken and it's
> broken since a lot now…

It's been broken (on this semicolon point) since it was born, because
the implementation was merged in #82 well before the spec landed in
#94.  That means that nobody has ever used it for spec-compliant
unpacking, which takes the pressure off (in my opinion) for fixing it
ASAP.